test/Java8andUp/playlist.xml

```diff
@@ -1843,5 +1843,46 @@
 			<subset>SE90</subset>
 		</subsets>
 	</test>
+	
+	<test>
+		<testCaseName>FileSystem-isAccessUserOnlyTests_SE90</testCaseName>
```

Comment:
For tests that should apply to Java9 and Up, its preferred to call the Java8 tests _SE80, and then drop the suffix, so the same test can be used for 9, 10, 11 and so on.